testing/xfd/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=xfd
+pkgver=1.1.2
+pkgrel=0
+pkgdesc="Xorg font display application."
+url="https://xorg.freedesktop.org/"
+arch="all"
+license="custom"
```

Comment:
This appears to use the [x11] license.

[x11]:https://spdx.org/licenses/X11.html